tests/ut_seattlelibv2_repypp_noargument.py

```diff
@@ -0,0 +1,38 @@
+#pragma out Invalid number of arguments
+#pragma out repypp.py infile outfile
+
+#pragma out preprocesses infile and includes content from the current directory.  Output is
+#pragma out written to outfile.   Outfile and infile must be distinct.
+
+"""
+ut_seattlelibv2_repypp_noargument.py -- This script tests if repypp.py check erroneous 
+#include statements(have no arguments at all).
```

Comment:
See above, docstring and test case (here: calling `repypp` with no arguments) don't match. Please resolve similarly.